MapboxNavigation/NavigationMapView.swift

```diff
@@ -56,6 +56,16 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
      */
     public weak var courseTrackingDelegate: NavigationMapViewCourseTrackingDelegate?
     
+    /**
+     The possible routes that can be represented on the map view.
+     */
+    public var routes: [Route]?
```

Comment:
I’m pretty uneasy about making this property public, because it’s essentially an implementation detail of `showRoutes(_:legIndex:)`, not something to be tampered with arbitrarily. If we can’t remove the dependency on this property from `CarPlayMapViewController.viewSafeAreaInsetsDidChange()`, then at least remove the documentation comment and use `:nodoc:` to hide the symbol from jazzy.